docs/src/tutorial/example-data.rst

```diff
@@ -0,0 +1,89 @@
+Run the workflow using example data
+===================================
+
+In this tutorial, you will run the workflow using example reference data.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`setup`. These instructions will install all of the software you need to complete this tutorial and others.
+
+Setup
+-----
+
+1. Activate the ``nextstrain`` conda environment:
+
+   .. code:: text
+
+      conda activate nextstrain
+
+2. Change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+3. Download the example tutorial repository into a new directory ``ncov-tutorial/``:
+
+   .. code:: text
+
+      $ git clone https://github.com/nextstrain/ncov-tutorial
```

Comment:
Minor point: could we drop the leading shell prompt `$` and example output to match the other `git clone` command in the setup page? It's handy when one can copy and paste an entire code block in the terminal without knowing to select the relevant subset.

Reply:
Done: 666ba683